lib/dry/system/plugins/bootsnap.rb

```diff
@@ -32,12 +32,12 @@ def self.dependencies
         def setup_bootsnap
           return unless bootsnap_available?
 
-          ::Bootsnap.setup(config.bootsnap.merge(cache_dir: root.join("tmp/cache").to_s))
+          ::Bootsnap.setup(**config.bootsnap.merge(cache_dir: root.join("tmp/cache").to_s))
         end
 
         # @api private
         def bootsnap_available?
-          RUBY_ENGINE == "ruby" && RUBY_VERSION >= "2.3.0" && RUBY_VERSION < "3.1.0"
+          RUBY_ENGINE == "ruby" && RUBY_VERSION >= "3.0.0" && RUBY_VERSION < "3.1.0"
```

Comment:
shouldn't the upper bound be removed? 🤔

Reply:
Oh sure, removed the `< 3.1.0` condition